packages/gestalt/src/Callout.js

```diff
@@ -0,0 +1,111 @@
+// @flow strict
+import * as React from 'react';
+import PropTypes, { string } from 'prop-types';
+import Box from './Box.js';
+import Icon from './Icon.js';
+import IconButton from './IconButton.js';
+import Link from './Link.js';
+import Text from './Text.js';
+import typography from './Typography.css';
+
+type Props = {|
+  description: string,
+    iconAccessibilityLabel: string,
+      primaryLink ?: {|
+        href: string,
+          label: string,
+  |},
+onDismiss: () => void,
+  secondaryLink ?: {|
+    href: string,
+      label: string,
+  |},
+style: 'error' | 'info' | 'warning',
+  title ?: string,
+|};
+
+const CALLOUT_TYPE_ATTRIBUTES = {
+  info: {
+    icon: 'check-circle',
+    color: 'blue',
+    backgroundColor: '#EBF4FE',
```

Comment:
@ashleyseto @anniemochi could we spec out what these colors should be in dark mode?

Reply:
At the moment, there is not a dark mode spec for this as it is a non-standard color so I would leave it identical in dark mode to ship.

Reply:
This is what currently happens in dark mode:
![image](https://user-images.githubusercontent.com/127199/88546811-c21cc000-cfd1-11ea-9028-535afb74e821.png)

So we do need to figure out a way forward. Right now the text colors and everything else automatically updates. The easiest way forward would be to have dark mode specific colors for these non-standard colors.

Reply:
Looks like more than just color isn't right in that above screenshot. My immediate recommendation since dark mode was not in scope when this was designed would be that it would look identical in dark and light mode much like primary buttons are the same in both.